frontend/components/campaigns/form/CampaignFormCharitySection.tsx

```diff
@@ -0,0 +1,48 @@
+import { Stack } from '@mui/system';
+import {
+  containerSx,
+  fieldsContainerSx,
+} from '../../../styles/components/campaigns/form/CampaignFormCharitySectionStyles';
+import { CharityAutocompleteData } from '../../../types/charity';
+import RemoveCircleOutlineIcon from '@mui/icons-material/RemoveCircleOutline';
+import FormTextInput from '../../forms/FormTextInput';
+import FormAutocomplete from '../../forms/FormAutocomplete';
+import IconButtonWithTooltip from '../../IconButtonWithTooltip';
+
+interface Props {
+  index: number;
+  charityOptions: CharityAutocompleteData[];
+  handleRemove: () => void;
+}
+
+const CampaignFormCharitySection = ({ index, charityOptions, handleRemove }: Props) => {
```

Comment:
Can the border be light gray?

Reply:
Wait how light? It looks light gray to me?

Reply:
Nvm not a big deal since this page is not user facing.